src/service/fake_generator.py

```diff
@@ -0,0 +1,35 @@
+import skimage
+import os
+import random
+
+from model.generator import Generator
+
+
+class FakeGenerator(Generator):
+    dir_path = "../../example/ai_gen_images/"
+    images = []
+
+    def generate(self, output_file_path):
+        """
+           Method that takes one random image of the ai_gen_image directory and puts it in another directory.
+        """
+        self.list_images()
+        random_image_index = random.randint(0, len(self.images) - 1)
+        random_image_path = self.images[random_image_index]
+        random_image_name = random_image_path.split("/")[-1]
+        random_image = skimage.io.imread(random_image_path)
+        os.mkdir(output_file_path)
+        skimage.io.imsave(output_file_path+random_image_name, random_image, check_contrast=False)
+
+    def list_images(self):
+        """
+            Helper method, that lists all images that are in the ai_gen_image directory.
+        """
+        for image in os.listdir(self.dir_path):
+            if os.path.isfile(os.path.join(self.dir_path, image)):
+                self.images.append(self.dir_path + image)
+
+
+# following is to test, TODO: can be deleted
+fakeGenerator = FakeGenerator()
+fakeGenerator.generate("../../fake_generator_images/")
```

Comment:
I would directly create a (py)test out of it.

Reply:
yes i thought that also, had no time. should have put it in description. is there a common way to create tests in python?, or just like in java with a test directory+matching subdirectories of src on the hight of src like this: 
src
    - service
        -fake_generator.py
test
    - service
        -fake_generator_test.py

Reply:
Yeah this is the way.

Reply:
Please also check first whether the directory already exists. This avoids the following error when executing the fake generator a second time. 
``
Traceback (most recent call last):
  File "C:\Daten\Universität\Wintersemester_23-24\Foundation_Models\Wettbewerb_InformatiCup\informatiCup2024\src\service\fake_generator.py", line 35, in <module>
    fakeGenerator.generate("../../fake_generator_images/")
  File "C:\Daten\Universität\Wintersemester_23-24\Foundation_Models\Wettbewerb_InformatiCup\informatiCup2024\src\service\fake_generator.py", line 21, in generate
    os.mkdir(output_file_path)
FileExistsError: [WinError 183] Eine Datei kann nicht erstellt werden, wenn sie bereits vorhanden ist: '../../fake_generator_images/'
``

Reply:
oh, will do that ty